zavertailoProj/src/Locators

```diff
@@ -1,6 +1,23 @@
 LoginPage
 
-button.login = ".//button........."// сам локатор
+button.login = ".//input[@id = 'username5']"
```

Comment:
Check this locator. Does it work every time? Remember about numbers ..

Reply:
.//input[@name="username" and @placeholder="Username"]